use validator::ValidationError;

pub fn validate_username(username: String) -> Result<(), ValidationError> {
    Ok(())
}

pub fn validate_password(password: String) -> Result<(), ValidationError> {
    Ok(())
}
